### DISCUSSION TITLE
Settings Manager Updates

### DISCUSSION THREAD

**Discussion by philipzae** [Xubuntu/xubuntu-default-settings#21] [Ideas]
Improvements are needed to be made to settings manager, like the inclusion of additional launchers and potentially the reorganization of entries into new categories/groups.

**Comment by philipzae** [comment]
### Launchers
New Launchers
- **Sound** - launches pavucontrol and would be in the Hardware category 
- **System Info** or **About System** - launches inxi or neofetch in a terminal window and would be in System or Hardware

Existing Launchers
- **Fonts** - would go in System category
- **GParted** - would go in Hardware category

### Categories

For the reorganization of categories, it would be good to adopt ideas from the organization of Gnome Settings, [Mate Control Center](https://ubuntu-mate.community/uploads/default/original/2X/8/889d76566ff5dd03535189f4e0bf8f68f6a387cf.jpg) and Cinnamon System Settings.

- All three have a section for customization of the desktop, including the background, theme, etc. Mate calls it _Look and Feel_ and Cinnamon calls it _Appearance_.
- Mate and Gnome have a section that separates connectivity items. Mate has network and firewall items in it, while Gnome has network and bluetooth items. Mate calls it _Internet and Network_.

**Additional Ideas**
- Might be useful to separate software management related items into a section
![image](https://user-images.githubusercontent.com/4412114/84734137-331f8f80-afb1-11ea-8237-af1bc3044ecc.png)
- We shouldn't have any entries by default in the others category

**Comment by philipzae** [comment]
Another place of inspiration for the organization of categories can be taken from Elementary OS.

![image](https://user-images.githubusercontent.com/4412114/86491605-2b673580-bd7c-11ea-81ca-60162c56437a.png)

**Comment by marcosins** [comment]
`Xfce Terminal` could be removed:

- Opens settings that the user is supposed to open from `xfce4-terminal`
- Follows Elementary System Settings, Mate Control Center and Cinnamon System Settings

Edit:
Alternatively, it could be renamed to **Terminal settings** or **Terminal preferences**, since `xcfe4-terminal` is the default terminal in Xubuntu.

![image](https://user-images.githubusercontent.com/14003389/87188798-7671d300-c2bd-11ea-9f6a-7287ba56876f.png)

**Comment by philipzae** [comment]
> `Xfce Terminal` could be removed:
>     * Opens settings that the user is supposed to open from `xfce4-terminal`
>     * Follows Elementary System Settings, Mate Control Center and Cinnamon System Settings

Thanks for the suggestion @marcosins . As we've hidden terminal-settings from the menu, hiding it in settings manager seems like a good idea as well.
https://bugs.launchpad.net/ubuntu/+source/xubuntu-default-settings/+bug/1851387

> Edit:
> Alternatively, it could be renamed to **Terminal settings** or **Terminal preferences**, since `xcfe4-terminal` is the default terminal in Xubuntu.

Changing it to 'Terminal Preferences' has already been suggested upstream.
https://gitlab.xfce.org/apps/xfce4-terminal/-/issues/50

**Comment by philipzae** [comment]
In addition to hiding Terminal Settings, hiding Bluetooth Adapters or Bluetooth Manager, seems like a good idea, so we only have a single entry for Bluetooth. I would suggest hiding Bluetooth Adapters, as you can access it from within Bluetooth Manager through Adapter > Preferences.

**Comment by philipzae** [comment]
Other insights can be taken from Mac OS X's System Preferences, though it resembles Elementary OS

![image](https://user-images.githubusercontent.com/4412114/87216510-7d4a2580-c351-11ea-9bbe-60813bb5ef8f.png)

Some differences in from this image with new updates ([image 1](https://www.askdavetaylor.com/3-blog-pics/apple-mac-system-preferences.png), [image 2](https://images.macrumors.com/article-new/2018/03/1-launch-system-preferences-macos.jpg?retina), [image 3](https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/Product_Help/en_US/PUBLIC_USERS/PL124/S0007_SystemPref.png))

- **Others** category
- **Startup Disk** moved to **Hardware**
- **App Store** in **Internet and Wireless** and replaces **Software Update** previously in **System**
- **Dictation and Speech** replaced by **Siri**
- **Wallet & Apple Pay** in **Internet and Wireless** and **Touch ID** in **System**

**Comment by philipzae** [comment]
> Mate and Gnome have a section that separates connectivity items. Mate has network and firewall items in it, while Gnome has network and bluetooth items. Mate calls it _Internet and Network_.

Elementary OS has the category as 'Network & Wireless' and Mac has it as 'Internet & Wireless' and this new category could be called 'Connectivity' and contain
- Advanced Network Configuration (we should likely rename this to Network)
- Bluetooth Manager (we should likely rename this to Bluetooth)
- Firewall Configuration (we should likely rename this to Firewall)
- Gigolo
- Gadmin Samba (we should likely rename this to Samba)

**Comment by marcosins** [comment]
Based on the defaults elements in 20.04
![image](https://user-images.githubusercontent.com/14003389/87319428-9bea2100-c4f7-11ea-9791-4990d8fd5c5d.png)

The modifications so far in this issue are:

## Look and Feel :new:
- Appearance
- Desktop
- Dock :new: (`plank --preferences`)
- ~~Menu Editor~~ -> Menu
- Panel
- Panel Profiles
- Qt5 Settings :information_source: (Not in default installation)
- Screensaver
- Window Manager
- Window Manager Tweaks
- Workspaces
- Xfdashboard settings :information_source: (Not in default installation)

## Personal
- About Me
- ~~Appearance~~ :x: 
- Backup :new:
- ~~Clipboard Manager Settings~~ :x: (Hidden)
- ~~Desktop~~ :x: 
- Language Support
- ~~Menu Editor~~ :x:
- Notifications
- ~~Panel~~ :x: 
- ~~Panel Profiles~~ :x: 
- ~~Preferred Applications~~ -> Default Applications
- ~~Screensaver~~ :x:
- ~~Window Manager~~ :x: 
- ~~Window Manager Tweaks~~ :x: 
- ~~Workspaces~~ :x:
- ~~Xfce Terminal~~ :x: (hidden)

## Software :new: 
- ~~Software~~ -> Software Store :new: 
- ~~Software Updater~~ -> Software Updates
- ~~Software & Updates~~ -> Software Sources
- Synaptic Package Manager :new: 

## Hardware
- ~~Additional Drivers~~ -> Drivers
- ~~Advanced Network Configuration~~ :x: 
- ~~Bluetooth Adapters~~ :x: (Hidden)
- ~~Bluetooth Manager~~ :x: 
- ~~Color Profiles~~ -> Color
- Disks :new:
- Display
- Keyboard
- Mouse and Touchpad
- NVidia X Server Settings :information_source: (Not in default installation)
- Power Manager
- Printers
- ~~Removable Drives and Media~~ -> Removable Storage and Devices
- Sound :new:

## Connectivity :new: 
- ~~Bluetooth Manager~~ -> Bluetooth
- ~~Firewall Configuration~~ -> Firewall :new: 
- ~~Advanced Network Connections~~ -> Network
- ~~Gigolo~~ -> Remote Connections :new:
- ~~Gadmin Samba~~ -> File Sharing :new: 

## System
- Accessibility
- Backup (System) :new:
- ~~MIME Type Editor~~
- Fonts :new: 
- IBus Preferences :information_source: (Not in default installation)
- ~~LightDM GTK+ Greeter settings~~ -> Login Window
- ~~Onboard Settings~~ -> Onscreen Keyboard
- ~~GParted~~ -> Partition Manager :new:
- Session and Startup
- Settings Editor
- ~~Software & Updates~~ :x: 
- ~~Software Updater~~ :x:
- System Info :new:
- Time and Date
- User and Groups

## ~~Other~~ :x:
- ~~Onboard Settings~~ :x: 
- ~~Settings Editor~~ :x:

---

I'll update this comment accordingly.

**Comment by marcosins** [comment]
My extra 2 cents in all of this:

- I can't clearly see the differences between **Personal** and  **Look and Feel** categories.
- **Additional Drivers** could be moved to **Software** too.
- **Software & Updates** could be renamed to **Software and Updates** to be consistent with other elements.
  - It could also be renamed to **Software and Repositories** because **Software Updater** and **Software and Updates** is confusing, even if they're totally different applications.
  - Software & Updates is the friendly name of `software-properties-gtk`, renaming it could be confusing if upon opening it the window title is indeed Software & Updates. It would be great to define how launcher names should relate to application names and how much the names could deviate from the original.
- **MIME Type Editor** is a little obscure IMO, it could be renamed to **Filetype Associations**
  - Windows groups Default apps and Associations under the same settings -> [Windows Settings](https://user-images.githubusercontent.com/14003389/87352300-d324f600-c528-11ea-8267-0d766de4f523.png)
  - Cinnamon/Linux Mint calls it **Preferred Applications** but doesn't allow individual mime type configuration -> [Cinnamon Settings](https://user-images.githubusercontent.com/14003389/87352827-b50bc580-c529-11ea-9e81-aeef3ee6d66f.png)
  - Mate calls it **Preferred Applications** too, but doesn't allow individual mime type configuration -> 
[Mate Control Center](https://user-images.githubusercontent.com/14003389/87353269-74607c00-c52a-11ea-9739-163edd172872.png)
  - Ubuntu Gnome calls it **Defaults Applications** but doesn't allow individual mime type configuration. It also has **Applications** where the user can set handlers for every application individually -> 
[Gnome Settings 1](https://user-images.githubusercontent.com/14003389/87354276-4bd98180-c52c-11ea-99c9-aa8ba5fe15b7.png) - [Gnome Settings 2](https://user-images.githubusercontent.com/14003389/87354531-ad015500-c52c-11ea-81c9-2666d5908545.png)
- Specific application names should be replaced by generic names.
  - Gparted -> Partition Editor (from the about dialog)
  - Gigolo -> Connect Filesystems (?)

**Comment by philipzae** [comment]
Thanks for doing that. Here are some changes.

- System Info/About System goes in **System** ('About' is used in Ubuntu, elementary, Mac OS, and 'About Xfce')
- Menu Editor, Panel, Panel Profiles, Window Manager, and Window Manager Tweaks would all go into **Look and Feel**
- move items from **Other** in **System**
- Add Gnome Software and Synaptic to **Software** as those will be new default installations

Found this organization from Ubuntu when it used to use Unity.
![image](https://user-images.githubusercontent.com/4412114/87355641-c24ca280-c571-11ea-9e87-ebd15238b00c.png)

**Comment by philipzae** [comment]
> * I can't clearly see the differences between **Personal** and  **Look and Feel** categories.

**Look and Feel** is for the appearance of the User Interface and **Personal** is for personal settings of the user.

> * **Additional Drivers** could be moved to **Software** too.

Selecting which driver to use for the hardware is still hardware related

> * **Software & Updates** could be renamed to **Software and Updates** to be consistent with other elements.

If its possible for us to rename any of them, we'll definitely have consistency.

>   * It could also be renamed to **Software and Repositories** because **Software Updater** and **Software and Updates** is confusing, even if they're totally different applications.

A better name would be great, but it does have multiple tabs for software repositories (Ubuntu Software, Other Software, Authentication) and multiple tabs for software updates (Updates, Developer Options, Livepatch). I presume the original name might have been **Software Repositories & Updates** and then shrunk it down to just **Software & Updates**. If it is in a category called software then we could easily call it **Repositories & Updates**.

>   * Software & Updates is the friendly name of `software-properties-gtk`, renaming it could be confusing if upon opening it the window title is indeed Software & Updates. It would be great to define how launcher names should relate to application names and how much the names could deviate from the original.

All the items we rename would likely have different window title labels, but the importance of the labels would be that they should be clear irrespective of where it takes you.

> * **MIME Type Editor** is a little obscure IMO, it could be renamed to **Filetype Associations**

Definitely would be better.

>   * Gparted -> Partition Editor (from the about dialog)

Likely **Partition Manager** is a more common wording

>   * Gigolo -> Connect Filesystems (?)

Not completely sure what this does, but yes a renaming would be good.

**Comment by marcosins** [comment]
Updated the comment with your changes.

---

> **Look and Feel** is for the appearance of the User Interface and **Personal** is for personal settings of the user.

Thanks for clarifying

> Selecting which driver to use for the hardware is still hardware related

Ok

> If its possible for us to rename any of them, we'll definitely have consistency.

All other elements in settings use **and** instead of **&**

> A better name would be great, but it does have multiple tabs for software repositories (Ubuntu Software, Other Software, Authentication) and multiple tabs for software updates (Updates, Developer Options, Livepatch). I presume the original name might have been **Software Repositories & Updates** and then shrunk it down to just **Software & Updates**. If it is in a category called software then we could easily call it **Repositories & Updates**.

The different launchers for different tabs of the same dialog is a mess, but I understand that this is more an Ubuntu inherited thing than xfce. **Repositories & Updates** sounds good to me

> All the items we rename would likely have different window title labels, but the importance of the labels would be that they should be clear irrespective of where it takes you.

Ok, get it

> Definitely would be better.

Nice!

> Likely **Partition Manager** is a more common wording

Sounds good to me too

> Not completely sure what this does, but yes a renaming would be good.

[_"It mounts what it is told to"_](https://www.uvena.de/gigolo/). It can connects to (s)ftp, samba, webdav and almost any kind of remote or local filesystem.

---
- Possible rename for Synaptic, because it's a more complex tool than Gnome Software:
  - Advance Software Manager

**Comment by philipzae** [comment]
> Updated the comment with your changes.

Thanks
 
> All other elements in settings use **and** instead of **&**

Yes Ubuntu changed all the **and**s to **&**s at one point, while Xfce still kept **and**s.

> The different launchers for different tabs of the same dialog is a mess, but I understand that this is more an Ubuntu inherited thing than xfce. **Repositories & Updates** sounds good to me

Mint Cinnamon has it as **Software Sources**.

> [_"It mounts what it is told to"_](https://www.uvena.de/gigolo/). It can connects to (s)ftp, samba, webdav and almost any kind of remote or local filesystem.

Maybe **Remote Connections**.

> * Possible rename for Synaptic, because it's a more complex tool than Gnome Software:

Synaptic shows up as **Synaptic Package Manager**.

### Additional Changes
- Order the categories alphabetically
- Remove 'Settings' in _Desktop Settings_ as that is the label in Settings Manager
- Rename _Removable Drives and Media_ to _Removable Storage and Devices_ ([submitted upstream](https://gitlab.xfce.org/xfce/thunar-volman/-/issues/7))
- Add _Workspaces_ to **Look and Feel**
- Add the renaming info to the _Network_ and _Bluetooth_ entries
- Rename _LightDM GTK+ Greeter settings_ to _Login Screen_
- Rename _MIME Type Editor_ to _Filetype Associations_
- You can remove '/About System' from 'System Info/About System' as final name will decided later
- Add new entry called _Backup (Deja-Dup)_ to **Personal**
- Add new entry _Disks_ to **Hardware**
- Add new entry called _Backup (System)_ to **System**
- Add '(Hidden)' after the X after 'Xfce Terminal' and 'Bluetooth Adapters'
- Add 'Clipboard Manager Settings X (Hidden)' into **Personal**, as this is one entry that will show up by default that we don't want to appear

### Placement of Non-Pre-installed Entries
Here are the list of entries that I have in my settings manager and where I would place them in the reordered categories
- Qt5 Settings -> Look and Feel
- Xfdashboard settings -> Look and Feel

For the ones in your screenshot
- NVidia X Server Settings -> Hardware
- IBus Preferences -> System

**Comment by marcosins** [comment]
> Maybe **Remote Connections**.

Renamed.

> Synaptic shows up as **Synaptic Package Manager**.

Renamed

> ### Additional Changes
> * Order the categories alphabetically
> * Remove 'Settings' in _Desktop Settings_ as that is the label in Settings Manager
> * Rename _Removable Drives and Media_ to _Removable Storage and Devices_ ([submitted upstream](https://gitlab.xfce.org/xfce/thunar-volman/-/issues/7))
> * Add _Workspaces_ to **Look and Feel**
> * Add the renaming info to the _Network_ and _Bluetooth_ entries
> * Rename _LightDM GTK+ Greeter settings_ to _Login Screen_
> * Rename _MIME Type Editor_ to _Filetype Associations_
> * You can remove '/About System' from 'System Info/About System' as final name will decided later
> * Add new entry called _Backup (Deja-Dup)_ to **Personal**
> * Add new entry _Disks_ to **Hardware**
> * Add new entry called _Backup (System)_ to **System**
> * Add '(Hidden)' after the X after 'Xfce Terminal' and 'Bluetooth Adapters'
> * Add 'Clipboard Manager Settings X (Hidden)' into **Personal**, as this is one entry that will show up by default that we don't want to appear

Updated. Also ordered items inside categories alphabetically.

> ### Placement of Non-Pre-installed Entries
> 
> Here are the list of entries that I have in my settings manager and where I would place them in the reordered categories
> 
> * Qt5 Settings -> Look and Feel
> 
> * Xfdashboard settings -> Look and Feel
> 
> For the ones in your screenshot
> 
> * NVidia X Server Settings -> Hardware
> 
> * IBus Preferences -> System

Added with a disclaimer: :information_source: (Not in default installation)

I'm aware of your [default apps document](https://docs.google.com/spreadsheets/d/1TiVcXNSfiDkwIIonz5WyBFAcTkg-hw7iIVzDH6_gsME) and but didn't know if they were officially addedd to the default installation since #5 is not marked as resolved yet.

**Comment by marcosins** [comment]
I see that `plank` is listed in your new apps document. If Plank is to  be included by default, the only way to configure it its by manually opening `plank --preferences`. Would be useful to include this launcher in **Look and Feel**. Elementary also uses `plank` but they call it Dock across their UI.

With this in mind, the  new entry should be called **Dock** or **Dock Preferences** if the Whisker Menu also includes a **Dock** launcher.

**Comment by philipzae** [comment]
Thanks for the changes. Here are some additional ones.
- you forgot the last ':' at the end of _Remote Connections_
- regarding the categories, I was mistaken about the order and it should be - Look and Feel, Personal, Software, Hardware, Connectivity, System
- 'Desktop Settings' wasn't being renamed to 'Desktop', 'Desktop' is its name in Settings Manager
- 'Synaptic' wasn't being renamed to 'Synaptic Package Manager', 'Synaptic Package Manager' is its name in Settings Manager
- 'Clipboard Manager Settings X' should be corrected having 'Clipboard Manager Settings' striken through and 'X' should be the remove image
- you forgot to add '(Hidden)' on the Xfce Terminal entry
- 'Software', which you have as 'Gnome Software' is to be renamed from 'Software' to ‘Software Store’
- 'Onboard Settings' can be renamed to 'Onscreen Keyboard'

> Added with a disclaimer: information_source (Not in default installation)

Nice

> I'm aware of your [default apps document](https://docs.google.com/spreadsheets/d/1TiVcXNSfiDkwIIonz5WyBFAcTkg-hw7iIVzDH6_gsME) and but didn't know if they were officially addedd to the default installation since #5 is not marked as resolved yet.

Just completed a new spreadsheet today in issue #5 to simplify what is hoped to get into the next release. I've shared it with the Xubuntu team for feedback and hope to finalize things this month if possible.
https://docs.google.com/spreadsheets/d/145vm29WnxgYm4Y_8guaGDAblZT5sRjZ3jhKRktgX1No/edit#gid=0

> I see that `plank` is listed in your new apps document. If Plank is to be included by default, the only way to configure it its by manually opening `plank --preferences`. Would be useful to include this launcher in **Look and Feel**. Elementary also uses `plank` but they call it Dock across their UI.
> 
> With this in mind, the new entry should be called **Dock** or **Dock Preferences** if the Whisker Menu also includes a **Dock** launcher.

If we ship it by default, hopefully we can add a **Dock** entry to **Look and Feel**.

**Comment by marcosins** [comment]
> - regarding the categories, I was mistaken about the order and it should be - Look and Feel, Personal, Software, Hardware, Connectivity, System

I was wandering this, every screenshot shared here uses a "user logical" order. Updated.

> - 'Desktop Settings' wasn't being renamed to 'Desktop', 'Desktop' is its name in Settings Manager

I don't know why **my** installation says Desktop Settings `¯\_(ツ)_/¯`. Fixed

> - 'Synaptic' wasn't being renamed to 'Synaptic Package Manager', 'Synaptic Package Manager' is its name in Settings Manager

Fixed

> - 'Clipboard Manager Settings X' should be corrected having 'Clipboard Manager Settings' striken through and 'X' should be the remove image

Too literal :man_facepalming: 

> - you forgot to add '(Hidden)' on the Xfce Terminal entry

Fixed

> - 'Software', which you have as 'Gnome Software' is to be renamed from 'Software' to ‘Software Store’

Fixed

> - 'Onboard Settings' can be renamed to 'Onscreen Keyboard'

Renamed

**Comment by philipzae** [comment]
Thanks. Was trying out Ubuntu Budgie and they renamed 'Software & Updates' to 'Software Sources'. :D

Here are some changes.
- _Network_ should be ~~Advanced Network Connections~~  -> _Network_
- _Preferred Applications_ and _MIME Type Editor_ have been merged into a single entry called Preferred Applications, so remove the MIME entry and rename the other to Default Applications

**Comment by marcosins** [comment]
> Thanks. Was trying out Ubuntu Budgie and they renamed 'Software & Updates' to 'Software Sources'. :D

Possible names so far:

- MacOS: Software Update
- Windows: Windows Update
- Elementary: About -> Check for Updates (more like software updater). They don't provide a GUI to configure repos. They're also promoting their own app store to download software and updates.
- Linux Mint 20: **Software Sources**
- Ubuntu Budgie: **Sofware Sources**
- Ubuntu Gnome: Software & Updates. Not included in settings,  is a standalone application
- Ubuntu Mate: Software & Updates


> * _Network_ should be ~Advanced Network Connections~  -> _Network_

Renamed!

> * _Preferred Applications_ and _MIME Type Editor_ have been merged into a single entry called Preferred Applications, so remove the MIME entry and rename the other to Default Applications

Renamed!. This is cool. Maybe i'll switch to a rolling release distro to check new xfce features.

**Comment by philipzae** [comment]
>     * MacOS: Software Update

This was replaced with 'App Store'

>     * Windows: Windows Update

This is similar to Software Updater and not a settings dialog.

>     * Linux Mint 20: **Software Sources**

Great that Mint is also using this naming, so let's go with that naming as well.

Please change the renaming to 'Samba' to 'File Sharing'.

**Comment by marcosins** [comment]
Updated.

I'm applying some of the renamings to my personal setup, looks pretty good imo.

![image](https://user-images.githubusercontent.com/14003389/87827859-b7339400-c849-11ea-9a07-4c2c1839025f.png)

**Comment by philipzae** [comment]
Didn't do the renaming yet but this is how it looks with the rearrangement.
![image](https://user-images.githubusercontent.com/4412114/87832012-00a9d080-c896-11ea-83e6-61f62693454d.png)

If you want to give it a go, extract these files into ~/.local/share/desktop-directories/
[desktop-directories.zip](https://github.com/Xubuntu/xubuntu-default-settings/files/4940389/desktop-directories.zip)

extract this file into /etc/xdg/xdg-xubuntu/menus/ (make sure to backup the file that's already there first)
[xfce-settings-manager.menu.zip](https://github.com/Xubuntu/xubuntu-default-settings/files/4940443/xfce-settings-manager.menu.zip)

extract these files into ~/.local/share/applications/ for the entries for **Sound** and **About System**
[applications.zip](https://github.com/Xubuntu/xubuntu-default-settings/files/4940413/applications.zip)

you'll also have to install deja-dup, timeshift, gufw, gadmin-samba, gnome-disk-utility, and synaptic.

**Comment by marcosins** [comment]
Thanks for the files, i'll check them out.

I was going to suggest adding `Personal -> File Manager Custom Actions` but it looks like you can't launch it by itself. Ideally this should exists just like `thunar -B`. I opened [#348 on gitlab](https://gitlab.xfce.org/xfce/thunar/-/issues/348) hoping that a developer implements it in time for `4.16` since one of the goals is to improve discoverability of this feature.

**Comment by philipzae** [comment]
I wouldn't add _File Manager Custom Actions_ to the Settings Manager, as its a setting of the file manager. Its the same reasoning not having the Xfce terminal settings.

**Comment by philipzae** [comment]
Some additional renaming
Additional Drivers -> Drivers
Software Updater -> Software Updates
Color Profiles -> Color
Menu Editor -> Menu

Mate has _Screensaver_ and _Notifications_ under **Look and Feel**, so lets do the same.
Mate has _GParted_ under **Administration**, so maybe we should have it in **System**. It would be better not to have 2 disk entries in Hardware and Disks is more user friendly and would be the better option for uses to use first.

Mint and Budgie use **Login Window**, so do you like that over **Login Screen**?

Add an 'Dock (`plank --preferences`)' entry under **Look and Feel**

You have IBus Preferences in your screenshot, are you utilizing that?

**Comment by marcosins** [comment]
> Additional Drivers -> Drivers

Updated :heavy_check_mark: 

> Software Updater -> Software Updates

Updated :heavy_check_mark: 

> Color Profiles -> Color

Updated :heavy_check_mark: 

> Menu Editor -> Menu

Updated :heavy_check_mark: 

> Mate has Screensaver and Notifications under Look and Feel, so lets do the same.

Updated :heavy_check_mark: 

> Mate has GParted under Administration, so maybe we should have it in System. It would be better not to have 2 disk entries in Hardware and Disks is more user friendly and would be the better option for uses to use first.

I agree that having `gparted` and `gnome-disks` is redundant. I think users that feels limited by the options provided by `gnome-disks` already know about `gparted` and can install it via `gnome-software`, `synaptic` or `apt`. Maybe `gparted` should be suggested in the installation slideshow and not shipped by default.

I also agree that this modification makes it clearer wich is the advanced one and that `gnome-disks` should be listed first. The name **Partition Manager** fits `gparted` well since partitions is the first thing the users see once opened and the list of drives is in a somewhat hidden menu, contrary to `gnome-disks` wich features every drive up front with big icons.

Updated :heavy_check_mark: (one in Hardware and the other in System)

> Mint and Budgie use Login Window, so do you like that over Login Screen?

Ubuntu Mate uses **Login Window** too. While I like Login Screen (because it the whole screen after all) I think it would be better to use the name present in other Ubuntu flavours/remixes to make Xubuntu part of the same family.

Updated :heavy_check_mark: (window)

> Add an 'Dock (`plank --preferences`)' entry under Look and Feel

Added :heavy_check_mark:

> You have IBus Preferences in your screenshot, are you utilizing that?

Not really. I checked and it was installed as a dependency of Zoom wich I have to use for work :man_shrugging:. On click it prompts me to start the daemon, so its just there using space.

**Comment by philipzae** [comment]
> Add an 'Dock (plank --preferences)' entry under Look and Feel
you forgot to add the bracket part, as a reference :D

Thanks for the updates. Spoke with Sean today about what it would take [to do the renaming](https://gitlab.xfce.org/xfce/xfce4-settings/-/issues/201) and he said he would look into it.

Regarding IBus, it seems its important for south east asian languages (chinese, japanese and korean), and I've suggested it be installed by default.

By the way, which icon theme are you using?

**Comment by marcosins** [comment]
> you forgot to add the bracket part, as a reference :D

Oops, added.

> Thanks for the updates. Spoke with Sean today about what it would take [to do the renaming](https://gitlab.xfce.org/xfce/xfce4-settings/-/issues/201) and he said he would look into it.

That's great!

**Comment by bluesabre** [comment]
Hey @philipzae and @marcosins, there's some great content here. I went ahead and converted it to a discussion since it seemed like a better fit. With everything listed so far, what remains to be added? Let's discuss it from here. :smile:

**Reply by philipzae** [reply]
1. being able to rename entries without having to create new ones, which you were to look into.
2. creating the new categories
3. adding / hiding the entries listed above, excluding the 'Sound' entry which has already been added in 21.04.
